WooCommerce/Classes/ViewModels/ProductDetailsViewModel.swift

```diff
@@ -797,4 +846,8 @@ extension ProductDetailsViewModel {
         static let weightUnit = "woocommerce_weight_unit"
         static let dimensionUnit = "woocommerce_dimension_unit"
     }
+
+    enum Constants {
```

Comment:
If the removal of the last new line is done using the method extracted to the helper class, maybe this is not necessary